open_lm/main.py

```diff
@@ -143,7 +143,15 @@ def load_optimizer(args, model, optimizer, scaler):
         logging.info(f"=> WARNING: not resuming optimizer.")
 
 
-def save_checkpoint(args, model, optimizer, scaler, completed_epoch, evaluation_loss):
+def load_data_chunk(args):
```

Comment:
maybe call this function `load_checkpoint`?

Reply:
I wanted to keep this separate since loading the chunk id only makes sense when loading without resampling (whereas the existing ```load_model``` handles the actual model loading. It might be clearer this way I believe.

Reply:
oh i see, u only load checkpoint to get the "data_chunk" field, makes sense then!